File: applications/cafe-scripted/scripts/python/barista.py
import time
import java

Drink = java.type("org.springframework.integration.samples.cafe.Drink")

def prepareDrink(orderItem):
	print("python: preparing %s for order %s" % (orderItem,orderItem.getOrder().getNumber()))
	time.sleep(eval(timeToPrepare))
	return Drink(orderItem.getOrder().getNumber(), orderItem.getDrinkType(), orderItem.isIced(),
              orderItem.getShots())
              
drink = prepareDrink(payload)          
              

File: applications/cafe-scripted/scripts/javascript/barista.js
importClass(org.springframework.integration.samples.cafe.Drink);
importClass(java.lang.Thread);
importClass(java.lang.System);
 
function prepareDrink(orderItem){ 
	Thread.sleep(timeToPrepare);
	System.out.println("javascript: preparing "+ orderItem + " for order "+ orderItem.getOrder().getNumber())
	return new Drink(orderItem.getOrder().getNumber(), orderItem.getDrinkType(), orderItem.isIced(),
            orderItem.getShots())
}

prepareDrink(payload);

